code-challenges.js

```diff
@@ -7,62 +7,142 @@
+// I set boilingPoint to 212 since that's what the instructions said was the boiling point.
+// From there, I began typing out the statement as if I were speaking to someone in a logical form: "if the temperature is less than the boiling point, then I want the outcome to say 'the temp is below the boiling point"
+// I established this same logic for all scenerios such as greater than and strictly equal to.
+
+// const temp = 42
 // Expected output: "42 is below boiling point"
+// let boilingPoint = 212
 
-// const temp = 350
+// if (temp < boilingPoint) {
+//     console.log(temp, "is below boiling point") 
+// } else if(temp > boilingPoint) {
+//     console.log(temp, "is above boiling point")
+// } else if(temp === boilingPoint) {
+//     console.log(temp, "is at boiling point")
+// }
+
+//  const temp = 350
 // Expected output: "350 is above boiling point"
+// let boilingPoint = 212
+
+// if (temp < boilingPoint) {
+//     console.log(temp, "is below boiling point") 
+// } else if(temp > boilingPoint) {
+//     console.log(temp, "is above boiling point")
+// } else if(temp === boilingPoint) {
+//     console.log(temp, "is at boiling point")
+// }
 
-// const temp = 212
+//  const temp = 212
 // Expected output: "212 is at boiling point"
+// let boilingPoint = 212
+
+// if (temp < boilingPoint) {
+//     console.log(temp, "is below boiling point") 
+// } else if(temp > boilingPoint) {
+//     console.log(temp, "is above boiling point")
+// } else if(temp === boilingPoint) {
+//     console.log(temp, "is at boiling point")
+// }
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Describe your process:
+// I knew that in needing to combine two arrays I was going to need an accessor since I don't need to change the original arrays, just combine them. 
+// I looked at the syllabus and remembered that .concat() does exactly that
+// In order to know that I was doing it correctly, I wrote out the code to combine the two arrays which was the first step.
+// The next step was that the output needed to be the length of the newly combined arrays. 
+// Since I knew that the .concat() code worked and knew the outcome, I established a new variable "padresBoth" which was the outcome of .concat()
+// I then used the .length property to communicate that I wanted the character count of padresBoth
+
+// const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+// const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+// // Expected output: 9
 
-const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
-const padres1998WorldSeriesRuns = [6, 3, 5, 3]
-// Expected output: 9
+// console.log(padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns))
+// let padresBoth = padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns)
+// console.log(padresBoth.length)
```

Comment:
Just a tip - you can string together methods, so you can actually add.length to the end of your let padresBoth = .... statement. and then just console.log(padresBoth)